subscriptions/views.py

```diff
@@ -1,26 +1,40 @@
-from rest_framework import viewsets, status
+from rest_framework import filters, viewsets, status
 from rest_framework.permissions import IsAuthenticated, IsAdminUser
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework.authtoken.models import Token
 from django.core.exceptions import ObjectDoesNotExist
 from django.contrib.auth.models import User
+import django_filters
 
 from .models import Subscription
 from .serializers import SubscriptionSerializer, CreateUserSerializer
 from .tasks import send_next_message, scheduled_metrics
 from seed_stage_based_messaging.utils import get_available_metrics
 
 
+class SubscriptionFilter(filters.FilterSet):
+    created_from = django_filters.IsoDateTimeFilter(
+        name="created_at", lookup_type="gte")
+    created_to = django_filters.IsoDateTimeFilter(
+        name="created_at", lookup_type="lte")
+
+    class Meta:
+        model = Subscription
+        filters = (
+            'identity', 'messageset_id', 'lang', 'active', 'completed',
```

Comment:
I think you need a filters list here, like this:
https://github.com/praekelt/seed-message-sender/blob/develop/message_sender/views.py#L71